python/cuml/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -258,8 +258,13 @@ class RandomForestClassifier(BaseRandomForestModel,
 
     _cpu_estimator_import_path = 'sklearn.ensemble.RandomForestClassifier'
 
+    _default_split_criterion = "gini"
+
     _hyperparam_interop_translator = {
-        "criterion": "NotImplemented",
+        "criterion": {
```

Comment:
Same comment as https://github.com/rapidsai/cuml/pull/6363/files#r1969774738 (we should translate the default value)